lib/logdna/writer.go

```diff
@@ -129,8 +131,12 @@ func extractToken(u *url.URL) (token string, err error) {
 
 func extractTags(u *url.URL, q url.Values) string {
 	tags := q["tag"]
-	list := make([]string, 0, len(tags))
+	if len(tags) == 0 {
+		tags = "";
+	}
+
+	list := make([]string, 0)
 	list = append(list, tags...)
 
-	return strings.Join(list, ",")
+	return fmt.Sprintf("tags=\"%s\"", strings.Join(list, ","))
```

Comment:
Potentially - this is my first time trying my hand at golang, so pardon my naiveté. If ```q["tag"]``` does not contain any tags or does not exist, does it default to null string ```""```?  This is important since our syslog parser (a fork of the glossy library) relies on having a null string for blank tags to be properly parsed out. If it defaults to null string ```""```, then I'll go ahead and shorten that function to that neat line you dropped in there :).